mappings/net/minecraft/entity/mob/PiglinEntityBrains.mapping

```diff
@@ -0,0 +1,83 @@
+CLASS net/minecraft/class_4838 net/minecraft/entity/mob/PiglinEntityBrains
+	FIELD field_22388 HUNT_MEMORY_TIME_RANGE Lnet/minecraft/class_4801;
+	FIELD field_22391 AVOID_MEMORY_TIME_RANGE Lnet/minecraft/class_4801;
+	FIELD field_22395 GOLDWARE Ljava/util/Set;
+	METHOD method_24713 hasZombifiedPiglinNearby (Lnet/minecraft/class_4836;)Z
+		ARG 0 piglin
+	METHOD method_24714 hasBeenHitByPlayer (Lnet/minecraft/class_4836;)Z
+		ARG 0 piglin
+	METHOD method_24719 wearsGoldArmor (Lnet/minecraft/class_1309;)Z
+		ARG 0 entity
+	METHOD method_24722 (Lnet/minecraft/class_4836;)V
+		ARG 0 piglin
+	METHOD method_24724 onAttacked (Lnet/minecraft/class_4836;Lnet/minecraft/class_1309;)V
+		ARG 0 piglin
+		ARG 1 attacker
+	METHOD method_24725 addCoreActivities (Lnet/minecraft/class_4836;Lnet/minecraft/class_4095;)V
+		ARG 0 piglin
+		ARG 1 brain
+	METHOD method_24731 drop (Lnet/minecraft/class_4836;Lnet/minecraft/class_1799;Lnet/minecraft/class_243;)V
+	METHOD method_24732 create (Lnet/minecraft/class_4836;Lcom/mojang/datafixers/Dynamic;)Lnet/minecraft/class_4095;
+		ARG 0 piglin
+	METHOD method_24733 onGoldBlockBroken (Lnet/minecraft/class_1657;)V
+		ARG 0 player
+	METHOD method_24735 isGoldWare (Lnet/minecraft/class_1792;)Z
```

Comment:
This name seems weird to me, maybe `isPlayerWearingGold`?

Reply:
Should've named it as `isGoldware`, including golden apple, carrot, melon, tools, etc.

Reply:
hmm, what about `isGolden`?

Reply:
`isGoldRelated` should be better

Reply:
Renamed to `isGoldenItem`